src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -220,8 +253,12 @@ static ucs_status_t uct_cuda_copy_mem_alloc(uct_md_h md, size_t *length_p,
     }
 
     if (!uct_cuda_base_is_context_active()) {
-        ucs_error("attempt to allocate cuda memory without active context");
-        return UCS_ERR_NO_DEVICE;
+        ucs_debug("attempt to allocate cuda memory without active context");
+        status = uct_cuda_copy_md_set_ctx(
+                ucs_derived_of(md, uct_cuda_copy_md_t));
```

Comment:
rename 'md' to 'tl_md' and add local var
uct_cuda_copy_md_t *md = ucs_derived_of(...)

Reply:
Done